rosdep/python.yaml

```diff
@@ -5465,6 +5465,17 @@ python-ujson:
     vivid:
       pip:
         packages: [ujson]
+python3-ujson:
```

Comment:
This key is not alphabetized. It should come just before `python3-unidiff`.

Reply:
confirmed, will relocate the pkg